src/main/java/com/solvd/pages/HomePage.java

```diff
@@ -0,0 +1,47 @@
+package com.solvd.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class HomePage {
+    WebDriver driver;
+
+    @FindBy(css = "a[class*='menu_home'][class*='active']")
```

Comment:
If you want to grab the home button by class, you can use @FindBy(css = "a.menu_home")